### PR TITLE
Fixes the termination condition for real time tests tab update. 

### PR DESCRIPTION
**Pull request by deepakaravindr**
Already, tested on TeamCity with actual functional tests build where
all the tests are run.

@alpaix @emgarten

### CODE REVIEW

**Comment by emgarten** [comment]
:shipit:

File: Scripts/NuGetFunctionalTestUtils.ps1 [hunk 1]
```diff
@@ -107,42 +107,40 @@ function RealTimeLogResults
                 }
 
                 $currentTestTime = 0
+                $currentTestId = $content.Count
+            }
+            else
+            {
+                Write-Host "Current Test is $($currentTestId) and current test time is $($currentTestTime)"
```
Comment (alpaix) [inline Scripts/NuGetFunctionalTestUtils.ps1:114]: Nit: or ${currentTestId} for shorter syntax

**Comment by alpaix** [comment]
:ship:

**Comment by deepakaravindr** [comment]
Merged with https://github.com/NuGet/NuGet.Client/commit/74c4c24b9f1fe3cb52a572d2844e6c7032923185

**Pull request closed without merging**